src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -70,6 +70,16 @@ public function doSomething(\Foo\Bar $foo): \Foo\Bar\Baz
         );
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        // should run after PhpdocToReturnTypeFixer
+        // should run before NoSuperfluousPhpdocTagsFixer
```

Comment:
the added test shows the prio issue for `NoSuperfluousPhpdocTagsFixer`, but not for `PhpdocToReturnTypeFixer`, is the priority test missing or is the comment a bit off?